shared/src/packages/projects/libs/types/project-search-request-dto.type.ts

```diff
@@ -0,0 +1,5 @@
+type ProjectSearchRequestDto = {
```

Comment:
```suggestion
type ProjectGetAllParametersDto = {
```
don't forget about file name